### PR TITLE
ls, mv, sudo: add pt_BR translation

### PR DESCRIPTION
**Pull request by OctavioFurio**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Comment by OctavioFurio** [comment]
Sorry for the many commits, I was having some trouble with formatting.
Thanks for the opportunity.

**Review by navarroaxel — CHANGES_REQUESTED** [review]
For the non-English pages we only accept translations in sync with the main English page. You can update this PR translating the examples from the following pages:
https://github.com/tldr-pages/tldr/blob/main/pages/common/ls.md
https://github.com/tldr-pages/tldr/blob/main/pages/common/mv.md
https://github.com/tldr-pages/tldr/blob/main/pages/common/sudo.md

**Comment by OctavioFurio** [comment]
Thank you for the instructions. I've just made the requested corrections.

File: pages.pt_BR/linux/ls.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# ls
+
+> Listar o conteúdo de um diretório. Mais informações [neste site](https://manned.org/ls).
+
+- Liste um arquivo por linha:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/ls.md:5]:
```suggestion
- Lista um arquivo por linha:
```

File: pages.pt_BR/linux/ls.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# ls
+
+> Listar o conteúdo de um diretório. Mais informações [neste site](https://manned.org/ls).
```
Comment (navarroaxel) [inline pages.pt_BR/linux/ls.md:3]:
```suggestion
> Listar o conteúdo de um diretório.
> Mais informações: <https://manned.org/ls>.
```

File: pages.pt_BR/linux/ls.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+- Liste um arquivo por linha:
+
+`ls -1`
+
+- Liste todos os arquivos, inclusive arquivos ocultos:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/ls.md:9]:
```suggestion
- Lista todos os arquivos, inclusive arquivos ocultos:
```

File: pages.pt_BR/linux/mv.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# mv
+
+> Movimentação de arquivos entre diretórios, ou renomeá-los. Mais informações [neste site](https://url-to-upstream.tld).
```
Comment (navarroaxel) [inline pages.pt_BR/linux/mv.md:3]:
```suggestion
> Movimentação de arquivos entre diretórios, ou renomeá-los.
> Mais informações: <https://www.gnu.org/software/coreutils/mv>.
```

File: pages.pt_BR/linux/mv.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+- Mover um arquivo para um diretório arbitrário:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/mv.md:5]:
```suggestion
- Move um arquivo para um diretório arbitrário:
```

File: pages.pt_BR/linux/mv.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+
+`mv {{arquivo}} {{destino}}`
+
+- Mover arquivos para outro diretório, mantendo os nomes dos arquivos:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/mv.md:9]:
```suggestion
- Move arquivos para outro diretório, mantendo os nomes dos arquivos:
```

File: pages.pt_BR/linux/mv.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+
+- Requisitar confirmação para sobrescrição de arquivos, independentemente das permissões de arquivo:
+
+`mv -i {{arquivo}} {{destino}}`
+
+- Não sobrescrever arquivos existentes no diretório de destino:
+
+`mv -n {{arquivo}} {{destino}}`
+
+- Mova os arquivos em modo Verbose, mostrando os arquivos após sua movimentação:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/mv.md:25]:
```suggestion
- Move os arquivos em modo Verbose, mostrando os arquivos após sua movimentação:
```

File: pages.pt_BR/linux/mv.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
+- Mover arquivos para outro diretório, mantendo os nomes dos arquivos:
+
+`mv {{arquivo_1}} {{arquivo_2}} {{arquivo_3}} {{destino}}`
+
+- Não requisitar confirmação para sobrescrição de arquivos:
+
+`mv -f {{arquivo}} {{destino}}`
+
+- Requisitar confirmação para sobrescrição de arquivos, independentemente das permissões de arquivo:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/mv.md:17]:
```suggestion
- Requisita confirmação para sobrescrição de arquivos, independentemente das permissões de arquivo:
```

File: pages.pt_BR/linux/mv.md [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+
+`mv -i {{arquivo}} {{destino}}`
+
+- Não sobrescrever arquivos existentes no diretório de destino:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/mv.md:21]: I think `sobrescritar` is the correct verb
```suggestion
- Não sobrescrita arquivos existentes no diretório de destino:
```

File: pages.pt_BR/linux/sudo.md [hunk 10]
```diff
@@ -0,0 +1,35 @@
+# sudo
+
+> Executa um único comando como o Superuser, ou como outro usuário. Para mais informações, acesse [este site](https://manned.org/sudo).
```
Comment (navarroaxel) [inline pages.pt_BR/linux/sudo.md:3]:
```suggestion
> Executa um único comando como o Superuser, ou como outro usuário.
> Mais informações: <https://www.sudo.ws/sudo.html>.
```

File: pages.pt_BR/linux/sudo.md [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+
+- Executar um comando como Superuser:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/sudo.md:5]:
```suggestion
- Executa um comando como Superuser:
```

File: pages.pt_BR/linux/sudo.md [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+
+`sudo {{less /var/log/syslog}}`
+
+- Editar um arquivo, como Superuser, com seu editor padrão:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/sudo.md:9]:
```suggestion
- Edita um arquivo, como Superuser, com seu editor padrão:
```

File: pages.pt_BR/linux/sudo.md [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+
+`sudo --edit {{/etc/fstab}}`
+
+- Executar um comando como outro usuário e/ou grupo:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/sudo.md:13]:
```suggestion
- Executa um comando como outro usuário e/ou grupo:
```

File: pages.pt_BR/linux/sudo.md [hunk 14]
```diff
@@ -0,0 +1,35 @@
...
+
+`sudo --user={{usuário}} --group={{grupo}} {{id -a}}}`
+
+- Executar um comando anterior com o prefixo `sudo` (apenas em `bash`, `zsh`, etc.):
+
+`sudo !!`
+
+- Abrir o shell padrão com privilégios de Superuser e executar arquivos de login (`.profile`, `.bash_profile`, etc.):
```
Comment (navarroaxel) [inline pages.pt_BR/linux/sudo.md:21]:
```suggestion
- Abre o shell padrão com privilégios de Superuser e executar arquivos de login (`.profile`, `.bash_profile`, etc.):
```

File: pages.pt_BR/linux/sudo.md [hunk 15]
```diff
@@ -0,0 +1,35 @@
...
+
+- Editar um arquivo, como Superuser, com seu editor padrão:
+
+`sudo --edit {{/etc/fstab}}`
+
+- Executar um comando como outro usuário e/ou grupo:
+
+`sudo --user={{usuário}} --group={{grupo}} {{id -a}}}`
+
+- Executar um comando anterior com o prefixo `sudo` (apenas em `bash`, `zsh`, etc.):
```
Comment (navarroaxel) [inline pages.pt_BR/linux/sudo.md:17]:
```suggestion
- Executa um comando anterior com o prefixo `sudo` (apenas em `bash`, `zsh`, etc.):
```

File: pages.pt_BR/linux/sudo.md [hunk 16]
```diff
@@ -0,0 +1,35 @@
...
+
+`sudo !!`
+
+- Abrir o shell padrão com privilégios de Superuser e executar arquivos de login (`.profile`, `.bash_profile`, etc.):
+
+`sudo --login`
+
+- Abrir o shell padrão com privilégios de Superuser sem alterar o ambiente de execução:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/sudo.md:25]:
```suggestion
- Abre o shell padrão com privilégios de Superuser sem alterar o ambiente de execução:
```

File: pages.pt_BR/linux/sudo.md [hunk 17]
```diff
@@ -0,0 +1,35 @@
...
+
+`sudo --shell`
+
+- Abrir o shell padrão como dado usuário, carregando o ambiente de execução deste usuário e lendo arquivos de login (`.profile`, `.bash_profile`, etc.):
+
+`sudo --login --user={{usuário}}`
+
+- Listar os comandos permitidos (e não permitidos) para o usuário atual:
```
Comment (navarroaxel) [inline pages.pt_BR/linux/sudo.md:33]:
```suggestion
- Lista os comandos permitidos (e não permitidos) para o usuário atual:
```

File: pages.pt_BR/linux/sudo.md [hunk 18]
```diff
@@ -0,0 +1,35 @@
...
+
+- Abrir o shell padrão com privilégios de Superuser e executar arquivos de login (`.profile`, `.bash_profile`, etc.):
+
+`sudo --login`
+
+- Abrir o shell padrão com privilégios de Superuser sem alterar o ambiente de execução:
+
+`sudo --shell`
+
+- Abrir o shell padrão como dado usuário, carregando o ambiente de execução deste usuário e lendo arquivos de login (`.profile`, `.bash_profile`, etc.):
```
Comment (navarroaxel) [inline pages.pt_BR/linux/sudo.md:29]:
```suggestion
- Abre o shell padrão como dado usuário, carregando o ambiente de execução deste usuário e lendo arquivos de login (`.profile`, `.bash_profile`, etc.):
```

**Review by navarroaxel — APPROVED** [review]

File: pages.pt_BR/linux/mv.md [hunk 19]
```diff
@@ -0,0 +1,28 @@
+# mv
+
+> Movimentação de arquivos entre diretórios, ou renomeá-los.
+> Mais informações: <https://www.gnu.org/software/coreutils/mv>
```
Comment (kbdharun) [inline pages.pt_BR/linux/mv.md:4]:
```suggestion
> Mais informações: <https://www.gnu.org/software/coreutils/mv>.
```

**Review by kbdharun — APPROVED** [review]

**Pull request merged**